Tests/Realm.Tests/Database/InstanceTests.cs

```diff
@@ -39,6 +39,23 @@ public void GetInstanceTest()
             GetRealm().Dispose();
         }
 
+        [Test]
+        public void ReadOnlyInstance_ThrowsOnRefresh()
+        {
+            var path = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "temp.realm");
+            var realm = Realm.GetInstance(path);
```

Comment:
This should use `GetRealm` instead which will take care of cleaning it up and deleting it.

Reply:
You can still dispose of the Realm even if you use `GetRealm`. Using GetRealm also ensures that the file gets deleted, even if the test fails.